solr/solrj/src/java/org/apache/solr/client/solrj/impl/LBHttpSolrClient.java

```diff
@@ -80,8 +81,8 @@ public class LBHttpSolrClient extends LBSolrClient {
   private final HttpSolrClient.Builder httpSolrClientBuilder;
   private volatile Set<String> urlParamNames = new HashSet<>();
 
-  private Integer connectionTimeout;
-  private volatile Integer soTimeout;
+  private Long connectionTimeoutMillis;
+  private volatile Long soTimeoutMillis;
```

Comment:
this does not need to be volatile anymore. it only comes from the builder there are no setters.